tasks/skylab/LoomUtils.wdl

```diff
@@ -17,7 +17,7 @@ task SmartSeq2LoomOutput {
     String pipeline_version
     Int preemptible = 3
     Int disk = 200
-    Int machine_mem_mb = 18
```

Comment:
18000 sounds like an odd value to request! I wonder if it was a typo

Reply:
Not sure. Happy to adjust it if you think another value is more suitable as a default

Reply:
Let's change it to 16000. I believe we will be charged for the machine that fits the cpu and memory and 18G is not standard. It should be either 16G or 32G